### ISSUE TITLE
Online normalization layer

### ISSUE THREAD

**Issue by DanielWicz** [tensorflow/addons#2532]
**Describe the feature and the current behavior/state.**
Online normalization layer allows for a normalization in the case, when input distribution changes significantly over time (RNN, online learning, GANs), and thus techniques like batch normalization hurt more than help.

**Relevant information**
- Are you willing to contribute it (yes/no):
*If you wish to contribute, then read the requirements for new contributions in [`CONTRIBUTING.md`](https://github.com/tensorflow/addons/blob/master/CONTRIBUTING.md#requirements-for-new-contributions-to-the-repository)*
- Are you willing to maintain it going forward? (yes/no): No
- Is there a relevant academic paper? (if so, where): https://arxiv.org/pdf/1905.05894.pdf (Accepted on NeurIPS 2019)
- Does the relavent academic paper exceed 50 citations? (yes/no): No, it's fresh and has 21
- Is there already an implementation in another framework? (if so, where): There is already an implementation for tensorflow: https://github.com/Cerebras/online-normalization together with a CPU and a GPU kernels, but does not work with the newest versions of TF (I think it works up to ~2.3).
- Was it part of tf.contrib? (if so, where): I don't know

**Which API type would this fall under (layer, metric, optimizer, etc.)**
Layer

**Who will benefit with this feature?**
People who use: Online learning, RNN, GANs

**Comment by AakashKumarNain** [comment]
> Does the relevant academic paper exceed 50 citations? (yes/no): No, it's fresh and has 21

At present, we are only accepting contributions that pass a minimum threshold. Please feel free to open this again once it meets the criteria
